libraries/libfc/src/crypto/signature.cpp

```diff
@@ -53,6 +53,20 @@ namespace fc { namespace crypto {
       }, _storage);
    }
 
+   bool signature::is_ecc_canonical() const {
```

Comment:
With recent changes, should this be renamed back to `is_canonical()`?

Reply:
haha yes will do.

Reply:
https://github.com/AntelopeIO/spring/pull/1708/commits/3b85a2184061016dbe3cd312f2238773963bb291